### PR TITLE
fav-movies.md: wonder woman 84 added at line 27

### PR DESCRIPTION
**Pull request by donAlexEc**
Hi, Alex.

I think Wonder Woman 84 is one of the best hero's movies of all time and should be part of this list.

Thanks a lot!

### CODE REVIEW

File: fav-movies.md [hunk 1]
```diff
@@ -24,3 +24,4 @@
 1. The Dark Knigth.
 2. Infinite war.
 3. Spiderman: No Way Home.
+4. Wonder Woman 84
```
[the remark this answers is not in the document]
Comment (donAlexEc) [reply fav-movies.md:27]: Sorry!

**Review by alexrated — CHANGES_REQUESTED** [review]
@donAlexEc please delete Wonder Woman 84. If you consider there is another movie that deserves to be in our list, then go ahead.

Thanks.

**Review by alexrated — APPROVED** [review]
Looks nice! @donAlexEc

**Comment by alexrated** [comment]
Everything is ok. Proceeding with merge.

**Pull request merged**